appengine/express-memcached-session/server.js

```diff
@@ -24,29 +24,33 @@ var app = express();
 
 app.use(cookieParser());
 app.use(session({
-    secret: 'appengineFTW',
-    key: 'test',
-    proxy: 'true',
-    store: new MemcachedStore({
-        hosts: [process.env.MEMCACHE_URL || '127.0.0.1:11211']
-    })
+  secret: 'your-secret-here',
+  key: 'view:count',
+  proxy: 'true',
+  store: new MemcachedStore({
+    hosts: [process.env.MEMCACHE_URL || '127.0.0.1:11211']
```

Comment:
Now that you slimmed down the readme, a little comment here about how `MEMCACHE_URL` is set by GAE would be helpful. :)